.vuepress/config.js

```diff
@@ -100,6 +100,13 @@ module.exports = {
                         '/community/tutorials/custom_theme_setup.md',
                     ],
                 },
+                {
+                  title: 'Advanced Usage',
+                  collapsable: false,
+                  children: [
+                    '/community/advanced_usage/artisan.md',
```

Comment:
This should just be under tutorials. There are a needless amount of sidebar sections. Can you also move node configuration to the tutorials section while you're at it?

Reply:
If you're talking about needless amount, shall I go ahead and move the creating and egg and docker image to tutorials as well? @DaneEveritt

Reply:
Nah, those kind of just fit where they are for now.